src/components/EditCoursePage/index.jsx

```diff
@@ -57,29 +57,39 @@ class EditCoursePage extends React.Component {
     return `edit-course-form-${this.props.courseInfo.data.uuid}`;
   }
 
-  prepareCourseRuns(courseRuns, courseData) {
-    /* eslint-disable no-param-reassign */
-    courseRuns.forEach((courseRun) => {
-      if (courseRun.staff) {
-        courseRun.staff = courseRun.staff.map(staffer => staffer.uuid);
+  prepareSendCourseRunData(courseData, targetRun) {
+    const sendCourseRuns = [];
+    const modifiedCourseRuns =
+      courseData.course_runs.filter(run => !IN_REVIEW_STATUS.includes(run.status));
+
+    // Reduce Staff list to just the UUID
+    modifiedCourseRuns.forEach((courseRun) => {
+      let draft = true;
+      if (targetRun && (courseRun.key === targetRun.key)) {
+        // If a course run triggered the submission, mark it as not a draft
+        draft = false;
       }
+      sendCourseRuns.push({
+        content_language: courseRun.content_language,
+        end: courseRun.end,
+        go_live_date: courseRun.go_live_date,
+        key: courseRun.key,
+        max_effort: courseRun.max_effort,
+        min_effort: courseRun.min_effort,
+        pacing_type: courseRun.pacing_type,
+        staff: courseRun.staff ? courseRun.staff.map(staffer => staffer.uuid) : courseRun.staff,
+        length: courseRun.length,
+        start: courseRun.start,
+        status: courseRun.status,
+        transcript_languages: courseRun.transcript_languages,
+        weeks_to_complete: courseRun.weeks_to_complete,
+        draft,
```

Comment:
Everything else is alphabetical, I feel like this should be too? Or is this a linting thing?